.github/labeler.yml

```
@@ -5,3 +5,5 @@ sig/operator:
 - 'operator/**/*'
 kind/feature:
 - 'operator/docs/enhancements/*'
+aread/helm:
```

I think there's a typo here
```suggestion
area/helm:
```